### ISSUE TITLE
Export /Import Profile

### ISSUE THREAD

**Issue by WeirDave** [Hekili/hekili#618]
Please describe your idea or feature and when it would be useful.
I'd like to be able to export my profile so I can load it in on my laptop when I am not able to play on my desktop. I can currently do this with ELVUI and your graphics model and detailed settings page seems to be loosely based on that so I thought I would suggest.

**Comment by Hekili** [comment]
Which settings are you wanting to share that you can't through Import/Export for your Displays?

The dirty secret of the addon is that all of the data for all of the specs is included in your profile.  So the export strings that includes 30 specializations' priorities is going to be pretty large.

**Comment by WeirDave** [comment]
I was primarily thinking of:

- Button Size
- Button Placement
- Number of buttons/steps

I like my stuff to be in the same spot for me where ever I play and whatever character/toon I use. 

THEN when I saw your message I just realized, and found, that I can export and import that stuff already under "Share Styles". At least that's what it looks like.

I just expected it to be under "Profile" like ELVUI is...

Sorry to have wasted your time. I do love your work and will be making a donation

-------------------
WeirDave
http://www.weirdave.com
